.travis.yml

```diff
@@ -0,0 +1,18 @@
+branches:
+  only:
+    - master
+cache:
+  directories:
+    - vendor
+sudo: required
+services:
+  - docker
+install:
+  - make bootstrap
+script:
+  - make test
+deploy:
+  provider: script
+  script: make ci-deploy
```

Comment:
The arguments for a separate deploy.sh would be:
- keeps the Makefile simpler
- that's what we're doing in deis/etcd, deis/workflow, deis/builder, deis/slugbuilder, and so on--they all have an identical _scripts/deploy.sh
- this is the quick-and-dirty "get an image up on Docker Hub" version, but very soon the logic will need to expand to include versioning and tags, and then we'll be glad it's consistently implemented
- hopefully we can stop the copy-paste and include deploy.sh from a central location at some point
- @sgoings thought this was the right initial approach